### DISCUSSION TITLE
Windows client

### DISCUSSION THREAD

**Discussion by Solamain** [advplyr/audiobookshelf#2056] [Ideas]
Was wondering if it was possible to make a stand alone client for a windows client like the one used for android.  There are times I want to listen to my audiobooks on my PC but do not want to deal with the browser tabs I always have saved for when I open my browser.

**Comment by CLHatch** [comment]
> Was wondering if it was possible to make a stand alone client for a windows client like the one used for android. There are times I want to listen to my audiobooks on my PC but do not want to deal with the browser tabs I always have saved for when I open my browser.

You can "sort of" do this already.  In Chrome, go to your ABS webpage.  Then go to `More Tools/Create Shortcut...`.  Pick a name you want for your "program", and check the `Open as window` checkbox.  Click on `Create` and it will create an icon on your desktop.  I pin this to my taskbar.

I'm pretty sure you can do the same in Firefox, but I'm not sure on the exact locations in the menu for that browser.

**Reply by Solamain** [reply]
Many thanks, can not believe I did not know about that, will make my life simpler in many ways :)

**Reply by emigrating** [reply]
Or you could just install the Android app and use that. Works great and looks just like a desktop app. (Of course, you'll have to be on Windows 11 and to make it easy build your own WSA image to allow for Google Play store, but works even as a sideload).

**Comment by CLHatch** [comment]
> Many thanks, can not believe I did not know about that, will make my life simpler in many ways :)

Took me a while to realize you could do this myself.  I actually pin a "Media" icon to my taskbar, which brings up my media server "landing page" running Organizr.  I have ABS in one of tabs on that page.
